lib/core/test/unit/Cardano/Wallet/Primitive/AddressDerivationSpec.hs

```diff
@@ -279,6 +292,23 @@ prop_passphraseHashMalformed
 prop_passphraseHashMalformed pwd = monadicIO $ liftIO $ do
     checkPassphrase pwd (Hash mempty) `shouldBe` Left ErrWrongPassphrase
 
+prop_keyToHexTextJcliCompatible
+    :: WalletKey k
+    => Unencrypted (k 'RootK XPrv)
+    -> Property
+prop_keyToHexTextJcliCompatible (Unencrypted k) = monadicIO $ do
+    let hexXPrv = B8.unpack . hex . unXPrvStripPub . getRawKey $ k
+    monitor (counterexample $ "\nkey bytes = " ++ hexXPrv)
+    (code, stdout, stderr) <- run $ jcliKeyFromHex hexXPrv
+    monitor (counterexample $ "\n" ++ show code)
+    monitor (counterexample $ "Stdout: " ++ show stdout)
+    monitor (counterexample $ "Stderr: " ++ show stderr)
+    assert (stderr == "")
+  where
+    jcliKeyFromHex = readProcessWithExitCode
+            "jcli"
+            ["key", "from-bytes", "--type", "ed25519bip32"]
```

Comment:
Another mysterious 8 character indent! :wink:

Reply:
Not sure, I might be affected by multiline let-bindings inside do-notation where 4-characters results in parse errors 🤷‍♂